project/build.scala

```diff
@@ -329,8 +329,8 @@ object Dependency {
     val Chaos = "0.8.7"
     val Guava = "19.0"
     // FIXME (gkleiman): reenable deprecation checks after Mesos 1.0.0-rc2 deprecations are handled
-    val MesosUtils = "1.0.2"
-    val Akka = "2.4.7"
+    val MesosUtils = "1.2.0"
```

Comment:
this is not guarded by feature-flag; implications is that future builds of Marathon depend on semantics of newer API, yes?